modules/metastore/src/Service.php

```diff
@@ -106,7 +106,7 @@ public function getSchema($identifier) {
    * @return \Drupal\metastore\Storage\MetastoreStorageInterface
    *   Entity storage.
    */
-  private function getStorage(string $schema_id): MetastoreStorageInterface {
+  public function getStorage(string $schema_id): MetastoreStorageInterface {
```

Comment:
I don't think we need to change the access level of this method. It seems like we could just call `DataFactory::getInstance()` directly in the tests for the NodeData class.

Reply:
You are correct. Changing.